File: Do_an_thi_trac_nghiem/java/ThiTracNghiem_DuAnCuoiKhoa/src/com/cybersoft/node/generic/Node.java
package com.cybersoft.node.generic;

public class Node<T> {
	public T data;
	public Node<T> next;

	public Node() {
		this.next = null;
	}

	public Node(T data) {
		this.data = data;
		this.next = null;
	}

}

File: Stack/c++/Stack/Node.cpp
#include <iostream> 
using namespace std;
template <typename T> class Node {
public:
	T data;
	Node<T> *next;

public:
	Node() {
	
		this->next = NULL;
	}

	 Node(T data) {

		this->data = data;
		this->next = NULL;
	}

};





